### DISCUSSION TITLE
[Feature Request]: 下载器整理根据下载路径判定是否进行整理

### DISCUSSION THREAD

**Discussion by FLIER001** [jxxghp/MoviePilot#3411] [讨论]
### 当前程序版本

1.4.6

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

建议开启下载器整理时，只整理在env.app中填写的文件路径中的媒体文件，因为平时还会下载一些音乐、电子书（现在全都会被识别整理）。

**Comment by kekylin** [comment]
现阶段可以通过搞一个MP专属下载器解决这个问题，像电子书那些通过其他下载器下载。
